test/unit/specs/data/find.js

```diff
@@ -28,10 +28,10 @@ describe('<Data> Find', function() {
       const result1 = await dataStore.find()
```

Comment:
Is there a test for running `dataStore.find()` with all the available parameters?

Reply:
No. I've tested them manually. I will create them

Reply:
ok, there should both `unit` and `e2e` tests

Reply:
\+